### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by jessicaschilling**
Added links to @johnnymatthews new contributor resources on docs beta site; disambiguated between docs legacy and docs beta

### CODE REVIEW

**Comment by jessicaschilling** [comment]
@johnnymatthews -- can you please review? Thanks!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -82,34 +86,34 @@ Sometimes small mistakes fall through through the cracks. So if you see an error
...
+Have an idea for a new page or article, or even a series of articles? The first step is to [create an issue](https://github.com/ipfs/docs/issues/new/choose) with your suggestions. The article you are suggesting may have been discussed before, or somebody else could already be working on it.
 
 Once an issue has been created, it can be discussed by the community and IPFS project leads. Once approved, you are free to start writing and implementing your documentation into the repository. Internal IPFS project leads from Protocol Labs review all pull-requests.
 
 ### File an issue
 
 Follow these steps to learn how to file an issue with the IPFS project.
 
-1. Go to the [Issues](https://github.com/ipfs/docs/issues/) tab in the [`ipfs/docs`](https://github.com/ipfs/docs) repository.
+1. Go to the [Issues](https://github.com/ipfs/docs/issues/) tab in the [`ipfs/docs`](https://github.com/ipfs/docs) repository. (Note: docs *issues* are tracked in `ipfs/docs`, but the *code and content* that power the docs beta site are in the `ipfs/ipfs-docs-v2` repo until the legacy docs site is deprecated -- currently slated for mid-2020.)
```
Comment (johnnymatthews) [inline CONTRIBUTING.md]: Could we take the _Note: docs *issues* are tracked_ section out of the brackets `()`? Auto-translators and screen readers can have a hard time with them, especially with longer sentences like this one.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -82,34 +86,34 @@ Sometimes small mistakes fall through through the cracks. So if you see an error
...
-5. Answer the questions listed in the issue form. Issues are created using Github Flavoured Markdown.
+5. Answer the questions listed in the issue form. Issues are created using Github Flavored Markdown.
 6. Once you have finished describing your issue, click **Submit new issue**.
 
 Shortly after your issue is submitted, an internal project lead from the IPFS project evaluates your issue. Further questions or requests-for-comment may be asked within the issue.
 
 ### Create a pull request
 
-If you have created some changes to a post, article, or entire section, you can submit a pull request to have it merged into the `ipfs/docs` repository.
+If you have created some changes to a post, article, or entire section, you can submit a pull request to have it merged into the `ipfs/ipfs-docs-v2` repository. (Note: docs *issues* are tracked in `ipfs/docs`, but the *code and content* that power the docs beta site are in the `ipfs/ipfs-docs-v2` repo until the legacy docs site is deprecated -- currently slated for mid-2020.)
```
Comment (johnnymatthews) [inline CONTRIBUTING.md]: Same as [above comment](https://github.com/ipfs/docs/pull/440/files#diff-6a3371457528722a734f3c51d9238c13R97):

> Could we take the Note: docs issues are tracked section out of the brackets ()? Auto-translators and screen readers can have a hard time with them, especially with longer sentences like this one.

File: CONTRIBUTING.md (hunk 1 above)
Comment (jessicaschilling) [reply CONTRIBUTING.md]: Done.

File: CONTRIBUTING.md (hunk 2 above)
Comment (jessicaschilling) [reply CONTRIBUTING.md]: Done.

**Comment by johnnymatthews** [comment]
Nice.

**Pull request merged**